QNetworkxGraph.py

```diff
@@ -1032,9 +1030,9 @@ def mousePressEvent(self, event):
                 self.setDragMode(QGraphicsView.ScrollHandDrag)
                 self.last_position = event.pos()
 
-        # If right button, avoid unselecting nodes not passing the event to the parent
-        if not event.buttons() & Qt.RightButton:
```

Comment:
Why we remove this? It was introduced to be able do right click on the background after several nodes are selected...